docs-v2/reference/api/integration/getLegacy.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Get an integration (Legacy)'
```

Comment:
Note Mintlify do not support `deprecated` property. I have sent them an email to see if they can change that

Reply:
can we have DEPRECATED in the title, instead of Legacy then.